third_party/nxp/tinycrypt/fixes/openthread_ecdsa_cpp.patch

```diff
@@ -71,7 +71,7 @@ index b66371d17..ac5460c28 100644
 +    Error                error;
 +    mbedtls_pk_context   pk;
 +    mbedtls_uecc_keypair *keyPair;
-+    int                  ret;
++//    int                  ret;
```

Comment:
Why do we need to comment the "ret" variable? Looks like it is already guarded MBEDTLS_USE_TINYCRYPT/

Reply:
Same as the other one, compilation with is_debug=true fails because it's more restrictive to errors. Variable ret is not used at all.